src/app/static/src/app/store/root/getters.ts

```diff
@@ -67,8 +67,14 @@ export const getters: RootGetters & GetterTree<RootState, RootState> = {
             modelOptions: filterWarnings(state.modelOptions.warnings, location),
             modelRun: filterWarnings(state.modelRun.warnings, location),
             modelCalibrate: filterWarnings(state.modelCalibrate.warnings, location),
-            reviewInputs: filterWarnings(state.genericChart.warnings, location)
+            reviewInputs: filterWarnings(spreadReviewInputsWarnings(state), location)
         }
     }
 }
 
+const spreadReviewInputsWarnings = (state: RootState) => {
```

Comment:
Might be a little more obvious?

```suggestion
const allReviewInputsWarnings = (state: RootState) => {
```